### PR TITLE
Add Order Independent Transparency example

### PR DESCRIPTION
**Pull request by daemyung**

### CODE REVIEW

**Comment by SaschaWillems** [comment]
Thank you very much for your PR. An OIT sample is something I always wanted to add, but never actually had the time to do so. So this is very much appreciated.

There are only a few small things that should be changed. I'll highlight them via separate comments.

File: examples/oit/oit.cpp [hunk 1]
```diff
@@ -0,0 +1,683 @@
+/*
+* Vulkan Example - High dynamic range rendering
+*
+* Note: Requires the separate asset pack (see data/README.md)
+*
+* Copyright by Sascha Willems - www.saschawillems.de
+*
+* This code is licensed under the MIT license (MIT) (http://opensource.org/licenses/MIT)
+*/
```
Comment (SaschaWillems) [inline examples/oit/oit.cpp:9]: This needs to be updated, as this refers to the HDR sample. Please also add your copyright below mine, so people can see that this sample was contributed b ypu.

File: examples/oit/oit.cpp [hunk 2]
```diff
@@ -0,0 +1,683 @@
...
+
+#include "vulkanexamplebase.h"
+#include "VulkanglTFModel.h"
+
+#define ENABLE_VALIDATION true
```
Comment (SaschaWillems) [inline examples/oit/oit.cpp:14]: This should be set to false by default

**Review by SaschaWillems — CHANGES_REQUESTED** [review]

File: examples/oit/oit.cpp (hunk 1 above)
Comment (daemyung) [reply examples/oit/oit.cpp]: Oh it's my mistake... I have changed typos and added copyright.

File: examples/oit/oit.cpp (hunk 2 above)
Comment (daemyung) [reply examples/oit/oit.cpp]: Done.

**Comment by daemyung** [comment]
> Thank you very much for your PR. An OIT sample is something I always wanted to add, but never actually had the time to do so. So this is very much appreciated.
> 
> There are only a few small things that should be changed. I'll highlight them via separate comments.

Thanks. I am happy to contribute. This implementation is very simple and performance is not good enough but

**Pull request merged**